### DISCUSSION TITLE
Tools

### DISCUSSION THREAD

**Discussion by anant-svc** [Greg0/Lazer-Database#71] [General]
In this discussion, we will let you know up coming tools for Lazer Database. That will help you in a unique way. Without changing the core files of Lazer database.

**Comment by anant-svc** [comment]
I was working on the migration tool for Lazer database. Though working on Unit testing was putting a delay to publish it.

Here is the migration tool - https://github.com/Lazer-Database/Migrate-to-Lazer-Database

You can install it via composer but it is not pushed to packagist yet.

**Reply by mitmelon** [reply]
Can this be used to convert MySQL database to Lazer.

2) It will really be good if encryption could be added as a feature to Lazer for encrypting the database.

What do you think.

**Reply by anant-svc** [reply]
Welcome @mitmelon Yes, this tool takes key/value pair arrays. i.e. [["a"=>1],["b"=>2]]
The LAZER_DATA_PATH constant should be defined before start then it will automatically create tables and save data.

**Reply by anant-svc** [reply]
@mitmelon Yes, encrypting data is very good for security and I will surely implement this feature.

**Comment by anant-svc** [comment]
@mitmelon @ddebin @reinos we have launched a new docs website - [lazer-database.pages.dev/](https://lazer-database.pages.dev/)
